app/models/series_azure.rb

```diff
@@ -0,0 +1,2 @@
+class SeriesAzure < Flavor
+end
```

Comment:
New line at end of file (please change your editor to just do this automatically)